pos_debranding/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2021 KMEE
+# License LGPL-3 or later (https://www.gnu.org/licenses/lgpl).
+
+{
+    "name": "Pos Debranding",
+    "summary": """Point of Sale: Remove Odoo SA Logo from POS""",
+    "version": "14.0.1.0.0",
+    "author": "KMEE, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/pos",
+    "license": "LGPL-3",
+    "maintainers": ["mileo"],
+    "depends": [
+        "point_of_sale",
+    ],
+    # Version 16 =D
+    # 'assets': {
+    #     'web.assets_qweb': [
+    #         'pos_debranding/static/src/xml/**/*',
+    #     ],
+    # },
```

Comment:
This can go away
```suggestion
```

Reply:
I migrate this module to version 16 first, sorry.